pkg/mcs/discovery/discover.go

```diff
@@ -25,6 +23,7 @@ import (
 	"github.com/tikv/pd/pkg/utils/etcdutil"
 	"go.etcd.io/etcd/clientv3"
 	"go.uber.org/zap"
+	"strconv"
```

Comment:
Please revert this change.

Reply:
Need to check why our static doesn't detect it here. 🤔